src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/DeleteApiDataSetVersionPlanViewModel.cs

```diff
@@ -0,0 +1,20 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using System;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+
+public record DeleteApiDataSetVersionPlanViewModel
+{
+    public Guid DataSetId { get; init; }
+
+    public string DataSetName { get; init; } = null!;
+
+    public Guid DataSetVersionId { get; init; }
```

Comment:
Similar to other PR comments, these could be updated to:

- `DataSet.Title`
- `Id`

Reply:
Yep, I did change this originally but then It got lost during rebasing. I've changed it again now, thanks! :)